_blueTheme.scss

```diff
@@ -483,4 +483,17 @@
             color: map-get($foreground, text);
         }
     }
+
+    /* Mat Table */
+    th.mat-header-cell {
+        color: map-get($foreground, text);
+    }
+    /* .mat-sort-header-arrow[style] {
+        opacity: 1!important;;
+        color: rgba(map-get($foreground, secondary-text), 1);
+    }
+    .mat-sort-header-sorted .mat-sort-header-arrow {
+        color: map-get($foreground, secondary-text);
+    }
+    */
```

Comment:
what are these comments for?

Reply:
Ah I should have taken these out; these start to match the Sortation icon styles that Suraj is working on.

Reply:
I'll remove them & update the dark theme color.

Reply:
Updated.